app/layout/project/sidebar/project/inventory-panel/components/inventory-table/index.tsx

```diff
@@ -43,25 +43,29 @@ const InventoryTable = ({
                   onChange={onSelectAll}
                 />
               </th>
-              <th className="flex-1 pl-2">
-                <HeaderItem
-                  text={'Name'}
-                  name={'name'}
-                  columns={columns}
-                  sorting={sorting}
-                  onClick={onSortChange}
-                />
-              </th>
-              <th className="flex flex-1 justify-start py-2 pl-14">
-                <HeaderItem
-                  className="justify-center"
-                  text={'Type'}
-                  name={'tag'}
-                  columns={columns}
-                  sorting={sorting}
-                  onClick={onSortChange}
-                />
-              </th>
+              {columns['name'] && (
```

Comment:
This approach will not scale. Can we pass an array of headers to the table an just render them? Like:

```
const headers = [
{
text: 'Name',
name: 'name',
columns,
sorting, 
onSortChange,
...,
{
...
}
]
```